### DISCUSSION TITLE
make it exceptionally clear that the directory listing for a server is indeed publicly visible to all

### DISCUSSION THREAD

**Discussion by ara4n** [element-hq/element-meta#831] [Ideas]
and that you should use groups, when ready, for private lists of groups of rooms

**Comment by andrewdunndev** [comment]
"groups when ready" is a hard one though...

**Comment by ara4n** [comment]
https://github.com/matrix-org/matrix-react-sdk/commit/bb0845954d5eeee5562d418174c8f43fa29f1460

**Comment by ara4n** [comment]
^ is a stopgap solution. will fix this properly when groups are here.
